fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirCodeSystemController.java

```diff
@@ -53,6 +77,106 @@
 		return CodeSystem.class;
 	}
 	
+	@PostMapping(consumes = { AbstractRestService.JSON_MEDIA_TYPE })
+	//@ResponseStatus(HttpStatus.CREATED)
+	public ResponseEntity<Void> create(@RequestBody final CodeSystem codeSystem) {
```

Comment:
I think this requires a pluggable interface where the individual tooling plug-ins can provide their CodeSystem create handler implementations. We cannot merge this into the 8.x branch, because it breaks the build.

Reply:
I agree. I have commented this code out.